src/cf-pagination/src/cf-pagination.less

```diff
@@ -16,10 +16,8 @@
 
 // Sizing variables
 
-@pagination-font-size-px:  @base-font-size-px;
-@pagination-font-size-em:  unit( @pagination-font-size-px / @base-font-size-px, em );
-@pagination-btn-height-px: 42px;
-@pagination-btn-width-px:  130px;
+@pagination-font-size-px:     @base-font-size-px;
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/463/commits/0b66aca860b6ce56b19bb5f0a6e0633ca63d7eb5